hidden_message.py

```diff
@@ -17,113 +17,20 @@
     L = O
 """
 
+# Set up the in and out values for the maketrans function
+in_values = 'ABCDEFGHIJKLabcdefghijklZYXWVUTSRQPOzyxwvutsrqpo'
+out_values = 'ZYXWVUTSRQPOzyxwvutsrqpoABCDEFGHIJKLabcdefghijkl'
+
+
 # Get user message
 message = str(input("Enter your message to encrypt here: "))
 
 
 # Replace the characters with encrypted characters    
 string_to_build = '' # create an empty string that we'll build with characters from the message
 for letter in message: # loops over each character in the message    
```

Comment:
You no longer need a loop, as the translate is performed on the entire function.  The current way you have this, you are running the maketrans and translate N times, where N is the length of the message someone entered.

Just remove the FOR and un-indent (dedent?) the code.

Reply:
OK thanks @kaltinril , I removed the unnecessary FOR loop